packages/scene-composer/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "scripts": {
     "build": "run-s compile",
+    "build:watch": "node ./tools/watch-build.js",
```

Comment:
Javascript conventions would be just name this script "watch" it's a very common operation.

Reply:
we have another `test:watch` command, so I'm trying to be consistent with that